assets/js/frontend/woocommerce.js

```diff
@@ -78,4 +78,17 @@ jQuery( function( $ ) {
 			}, 1000 );
 		}
 	};
+
+	// Show password visiblity hover icon on woocommerce forms
+	$('.woocommerce form #password').wrap('<span class="password-input"></span>');
```

Comment:
This code isn't following the WordPress coding standards, see https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#spacing

Reply:
Woops - first timer here :)

I reviewed those standards and I believe I'm within spec now.